Unable to read res files with mdbtools>=1.0

Using mdbtools v1.0 with galvani leads to the following error when parsing .res files:

```python    
s3db = <sqlite3.Connection object at 0x7fd2bc033840>
filename = '/home/runner/work/navani/navani/tests/../Example_data/arbin_example.res'
table = 'Event_Table'

    def mdb_get_data_text(s3db, filename, table):
        print("Reading %s..." % table)
        insert_pattern = re.compile(
            r'INSERT INTO "\w+" \([^)]+?\) VALUES \(("[^"]*"|[^")])+?\);\n',
            re.IGNORECASE
        )
        try:
            # Initialize values to avoid NameError in except clause
            mdb_output = ''
            insert_match = None
            with sp.Popen(['mdb-export', '-I', 'postgres', filename, table],
                          bufsize=-1, stdin=sp.DEVNULL, stdout=sp.PIPE,
                          universal_newlines=True) as mdb_sql:
    
                mdb_output = mdb_sql.stdout.read()
                while len(mdb_output) > 0:
                    insert_match = insert_pattern.match(mdb_output)
>                   s3db.execute(insert_match.group())
E                   AttributeError: 'NoneType' object has no attribute 'group'

../../../.local/share/virtualenvs/navani-KXBeI0hV/lib/python3.10/site-packages/galvani/res2sqlite.py:455: AttributeError
```


On Ubuntu 22.04, mdbtools v1 is now the default version (whereas it capped out at 0.7 on 20.04). I'm happy to investigate this failure in detail on the mdbtools side and see if there is an easy way to add support for both versions in this repo, but if someone sees this first and knows what is going on, feel free to take over! 

You can see this clearly in the status checks of https://github.com/the-grey-group/navani/pull/4, where all I am changing in the last few commits is the base image of our CI.

As a side note, it would be nice to have CI running in this repo again, as it still seems to have active use. I'm happy to set this up on GitHub actions as I can't tell if Travis is still running anywhere. Probably this would need a maintainer of the repository to enable the workflows for this repo (I will make a separate issue).

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -17,8 +17,7 @@ jobs:
 
   pytest:
     name: Run Python unit tests
-    # Note that 20.04 is currently required until galvani supports mdbtools>=1.0.
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
 
     strategy:
       fail-fast: false

--- a/galvani/res2sqlite.py
+++ b/galvani/res2sqlite.py
@@ -439,7 +439,8 @@ CREATE VIEW IF NOT EXISTS Capacity_View
 def mdb_get_data_text(s3db, filename, table):
     print("Reading %s..." % table)
     insert_pattern = re.compile(
-        r'INSERT INTO "\w+" \([^)]+?\) VALUES \(("[^"]*"|[^")])+?\);\n', re.IGNORECASE
+        r"""INSERT INTO "\w+" \([^)]+?\) VALUES (\((('[^']*')|"[^"]*"|[^')])+?\),?\s*)+;\n""",
+        re.IGNORECASE,
     )
     try:
         # Initialize values to avoid NameError in except clause

